coding-guides/using-audit-logs.md

```diff
@@ -0,0 +1,142 @@
+# Understanding Audit Logs
+
+From time to time, users in "An Idiot's Guide Official Server" need to reference audit logs for whatever reason;
+let it be viewing them for certain actions to adding things into the audit logs. 
+This guide will explain everything about audit logs and how to use them.
+
+The first thing that you will need is a working Discord Bot. If you do not have one, please visit [Your First Bot](getting-started/your-basic-bot.md) to get started.
+
+Now, some things to take note of. This guide is using discord.js@11.3.2 The bot will need some permissions. The main permission the bot will need is `'VIEW_AUDIT_LOGS'`. This permission allows the bot to view the audit logs and do whatever you want to, obeying the Discord ToS that is. 
```

Comment:
I feel like "do whatever you want, obeing the ToS" offers no significant information for the reader. This sentence can end at `view the audit logs.`